crates/core_arch/src/x86/sse.rs

```diff
@@ -1453,6 +1453,31 @@ pub unsafe fn _mm_storer_ps(p: *mut f32, a: __m128) {
     *(p as *mut __m128) = b;
 }
 
+/// Store 16-bit integer from the first element of `a` into memory.
+///
+/// `mem_addr` does not need to be aligned on any particular boundary.
+///
+/// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_storeu_si16)
+#[inline]
+#[target_feature(enable = "sse")]
+//#[stable(feature = "simd_x86", since = "1.45.2")]
```

Comment:
Please add stability attributes.

Reply:
Unstable, a `feature` name like `"simd_x86_2"` is fine. It does not need a reason.